docs-client/src/containers/App/index.tsx

```diff
@@ -191,7 +192,14 @@ const AppDrawer: React.FunctionComponent<AppDrawerProps> = ({
                   onClick={() => handleServiceCollapse(service.name)}
                 >
                   <ListItemText>
-                    <Typography variant="subtitle1">
+                    <Typography display="inline" variant="body2">
+                      <code>
+                        {specification.hasUniqueServiceNames()
+                          ? ''
+                          : `${packageName(service.name)}.`}
+                      </code>
+                    </Typography>
+                    <Typography display="inline" variant="subtitle1">
                       <code>{simpleName(service.name)}</code>
```

Comment:
Would it be worth showing the full name when the mouse hovers when it has unique service names? 😆

Reply:
Added
![image](https://user-images.githubusercontent.com/8417787/146187941-0fff54ad-c2d3-4d84-b036-0c30e7c5277b.png)

Reply:
@minwoox Would it be better if we show tooltip over struct/enum names as well?

Reply:
Yeah, I think it's useful. 😆

Reply:
@minwoox Please review https://github.com/line/armeria/pull/3979/commits/1ac876a7838a249e86646c630410ffe55d1a2ae8 again. I fixed a bug where the tooltip of the service name shows regardless whether there are unique names or not, and added tooltip for enum and struct. 🙇

Reply:
Thanks! Still LGTM. 👍